benefits/in_person/context.py

```diff
@@ -0,0 +1,26 @@
+from dataclasses import dataclass, asdict
+
+from benefits.core.context import SystemName
+
+
+@dataclass
+class EligibilityIndex:
+    policy_details: str
+
+    def dict(self):
+        return asdict(self)
+
+
+eligibility = {
+    SystemName.OLDER_ADULT.value: EligibilityIndex(
+        policy_details="I confirmed this rider’s identity using a government-issued ID and verified they are age 65 or older."
+    ),
+    SystemName.MEDICARE.value: EligibilityIndex(
+        policy_details="I confirmed this rider’s identity using a government-issued ID and verified they posses a valid "
```

Comment:
Copied this from Figma. Just noticed the spelling error in `posses` - should be `possess`